Docker/SolrServer/docker-entrypoint-initdb.d/disable-cores.sh

```diff
@@ -0,0 +1,13 @@
+#/bin/bash
+
+if [ ! -z "${TYPO3_SOLR_ENABLED_CORES}" ]
+then
+    mv /var/solr/data/cores /var/solr/data/cores_disabled
+    mkdir /var/solr/data/cores
+    for CORE in ${TYPO3_SOLR_ENABLED_CORES}
+    do
+        echo "Enable core ${CORE}"
+        mv /var/solr/data/cores_disabled/${CORE} /var/solr/data/cores/${CORE}
+    done
+    rm -rf /var/solr/data/cores_disabled
+fi
```

Comment:
I would suggest to use the version with possibility to enable additional cores.

```suggestion
#!/usr/bin/env bash

if [ -n "${TYPO3_SOLR_ENABLED_CORES}" ]
then
  [ -d /var/solr/data/cores_disabled ] || mkdir /var/solr/data/cores_disabled
  mv /var/solr/data/cores/* /var/solr/data/cores_disabled/.
  for CORE_TO_DISABLE in /var/solr/data/cores_disabled/*/core.properties
  do
    mv "${CORE_TO_DISABLE}" "${CORE_TO_DISABLE}_disabled";
  done

  for CORE_to_ENABLE in ${TYPO3_SOLR_ENABLED_CORES}
  do
    echo "Enable core ${CORE_to_ENABLE}"
    mv "/var/solr/data/cores_disabled/${CORE_to_ENABLE}" "/var/solr/data/cores/${CORE_to_ENABLE}"
    mv "/var/solr/data/cores/${CORE_to_ENABLE}/core.properties_disabled" "/var/solr/data/cores/${CORE_to_ENABLE}/core.properties"
  done
fi
```